chicago/permit_cleaning.py

```diff
@@ -21,21 +21,21 @@
     - At the beginning of each year: update year to current year in SQL_QUERY inside pull_existing_pins_from_athena() function
```

Comment:
All changes to this file are the result of passing it through the ruff formatter. I did not make any changes to the logic beyond running the formatter.